cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -523,6 +526,30 @@
     },
     "system_details": {
       "properties": {
+        "container_images": {
+          "items": {
+            "properties": {
+              "description": {
+                "description": "A brief description of the container image",
+                "type": "string"
+              },
+              "recid": {
+                "description": "The internal ID of the container image, if it is another Open Data record",
+                "type": "string"
+              },
+              "registry": {
+                "description": "The type of registry the image can be found at",
```

Comment:
It may be useful to provide example values in the documentation:

- Registry type where the image can be found (e.g. dockerhub, cerngitlab)